packages/story-editor/src/utils/getInsertedElementSize.ts

```diff
@@ -24,28 +24,27 @@ import {
   DEFAULT_DPR,
 } from '@googleforcreators/units';
 import { getDefinitionForType } from '@googleforcreators/elements';
+import type {
+  Element,
+  ElementType,
+  Direction,
+} from '@googleforcreators/elements';
+import type { Resource } from '@googleforcreators/media';
 
-const RESIZE_WIDTH_DIRECTION = [1, 0];
+const RESIZE_WIDTH_DIRECTION: Direction = [1, 0];
 
-/**
- * @param {?number|undefined} value The value.
- * @return {boolean} Whether the value has been set.
- */
-function isNum(value) {
+function isNum(value: unknown): value is number {
   return typeof value === 'number';
 }
 
 function getInsertedElementSize(
-  type,
-  width,
-  height,
-  attrs,
+  type: ElementType,
+  width: number,
+  height: number,
+  attrs: object,
   ratio = 1,
-  resource = null
+  resource: Resource
```

Comment:
```suggestion
  resource?: Resource
```

Since it was basically optional before and a resource is not always provided (for example `useInsertPreset` does not pass a resource)

Reply:
Fixed in https://github.com/GoogleForCreators/web-stories-wp/pull/12737/commits/6fd2808d4c2c96bf41c8807940aa91c85753e4dd